solution_runner/commands/setup_command.py

```diff
@@ -0,0 +1,130 @@
+from datetime import datetime
+from pathlib import Path
+
+import click
+import requests
+
+from . import consts
```

Comment:
```suggestion
import consts
```

Reply:
I wish it would work, the import didn't work otherwise. Maybe when I'm finished I'll come back and make the imports more elegant.